src/meta.jl

```diff
@@ -240,7 +240,8 @@ docstring(pc::PolymakeCallable) = get(pc.json, "doc", "")
 
 docstring(obj::PolymakeObject) = get(obj.json, "help", "")
 istemplated(obj::PolymakeObject) = haskey(obj.json, "params")
-templates(obj::PolymakeObject) = Symbol.(get(obj.json, "params", String[]))
+mandatorytemplates(obj::PolymakeObject) = get(obj.json, "mandatory_params", 0)
```

Comment:
this is just a nitpick (so you can ignore this:)
I'm not particularly happy with the name: `templates` gives you a vector (now: dict) of template names, whereas `mandatorytemplates` gives an Int.

but I don't have better suggestion either :)